src/qililab/platform/platform.py

```diff
@@ -655,6 +658,11 @@ def compile(self, program: PulseSchedule | Circuit, num_avg: int, repetition_dur
 
         programs = {}
         for pulse_bus_schedule in pulse_schedule.elements:
+            if len(pulse_bus_schedule.timeline) != 0:  # can't do only one conditional if list is empty
```

Comment:
Will the list ever be empty??

Reply:
If so, shouldn't we list an error when it is empty?

Reply:
it's empty for the flux schedules if there are no flux pulses. I believe we haven't fixed that yet.